jenkins/Makefile

```diff
@@ -69,13 +66,13 @@ push-rpmbuild-all: $(RPMBUILD_BASEIMAGE_DEFAULTS:%=push-rpmbuild-%)
 image-rpmbuild-all: $(RPMBUILD_BASEIMAGE_DEFAULTS:%=image-rpmbuild-%)
 
 push-ci:
-	buildah push localhost/pbench-ci-fedora:${BRANCH} quay.io/pbench/pbench-ci-fedora:${BRANCH}
+	buildah push quay.io/pbench/pbench-ci-fedora:${BRANCH}
 
 image-ci: ci.fedora.Dockerfile
-	buildah bud -f ci.fedora.Dockerfile -t localhost/pbench-ci-fedora:${BRANCH}
+	buildah bud -f ci.fedora.Dockerfile -t quay.io/pbench/pbench-ci-fedora:${BRANCH}
 
 ${_DISTROS:%=push-rpmbuild-%}: push-rpmbuild-%:
-	buildah push localhost/pbench-rpmbuild:${*} ${RPMBUILD_IMAGE_REPO}/pbench-rpmbuild:${*}
+	buildah push ${PB_CONTAINER_REG}/pbench-rpmbuild:${*}
```

Comment:
We're using a parameter for the `push-rpmbuild-%` target; should we be using a similar parameter (or another parameter of the same name?) in the `push-ci` and `image-ci` targets?  (I.e., if we're purging literal references to `images.paas.redhat.com`, should we be doing the same for references to `quay.io`?)

Reply:
Anybody can access quay.io to pull images, and if they want to participate in building, they can get their own account and be added to the org.  So I don't think there is a need to purge the literal references at this point (that is to say, not a change worth making in this PR).

If we want somebody to use another registry entirely, that would be great as a follow-on PR.